usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/whiteboardReset.js

```diff
@@ -87,13 +89,29 @@
             }
             var position = Entities.getEntityProperties(_this.entityID, 'position').position;
             _this.playSound(RESET_SOUND, RESET_SOUND_VOLUME, MyAvatar.position, true, false);
-            Entities.findEntitiesByName("Whiteboard Polyline", position, SEARCH_RADIUS_M).
-                forEach(function(whiteboardPiece) {
-                    var name = Entities.getEntityProperties(whiteboardPiece, 'name').name;
-                    if (name && name === "Whiteboard Polyline") {
-                        Entities.deleteEntity(whiteboardPiece);
+            var polylines = Entities.findEntitiesByName("Whiteboard Polyline", position, POLYLINES_SEARCH_RADIUS_M);
```

Comment:
this seems like a roundabout solution.

is there a reason why the polyline entities aren't parented to the whiteboard? it seems like that would solve two problems:
1. the one we're running into, because then this code could look at the whiteboard's children and delete all child polyline entities
2. as soon as someone tries to move a whiteboard and they notice that the whiteboard lines don't follow the whiteboard but the controls do, someone will file a new bug

thoughts?